onnxruntime/python/tools/quantization/operators/softmax.py

```diff
@@ -80,6 +80,11 @@ def quantize(self):
         if self.quantizer.activation_qType == onnx.onnx_pb.TensorProto.UINT8:
             out_scale = 1 / 256.0
             out_zero_point = 0
+        elif self.quantizer.is_activation_symmetric:
+            # results are all greater or equal to 0, so we can only use
+            # half of the range
+            out_scale = 1 / 127.0
```

Comment:
Would it be 128.0? as we have closed interval [0,127]

Reply:
the symmetric range is [-127,127], have to discard -128 to make is symmetric. This range has 254 unit intervals, half of it is 127 intervals. So the correct value is 1/127